examples/mujoco/td3/train_td3.py

```diff
@@ -0,0 +1,202 @@
+
+    args.outdir = experiments.prepare_output_dir(
+        args, args.outdir, argv=sys.argv)
+    print('Output files are saved in {}'.format(args.outdir))
+
+    # Set a random seed used in ChainerRL
+    misc.set_random_seed(args.seed, gpus=(args.gpu,))
+
+    def make_env(test):
+        env = gym.make(args.env)
+        # Unwrap TimeLimit wrapper
+        assert isinstance(env, gym.wrappers.TimeLimit)
+        env = env.env
+        # Use different random seeds for train and test envs
+        env_seed = 2 ** 32 - 1 - args.seed if test else args.seed
+        env.seed(env_seed)
+        # Cast observations to float32 because our model uses float32
+        env = chainerrl.wrappers.CastObservationToFloat32(env)
+        if args.monitor:
+            env = gym.wrappers.Monitor(env, args.outdir)
+        if args.render and not test:
+            env = chainerrl.wrappers.Render(env)
+        return env
+
+    env = make_env(test=False)
+    timestep_limit = env.spec.tags.get(
+        'wrapper_config.TimeLimit.max_episode_steps')
+    obs_space = env.observation_space
+    action_space = env.action_space
+    print('Observation space:', obs_space)
+    print('Action space:', action_space)
+
+    action_size = action_space.low.size
+
+    winit = chainer.initializers.LeCunUniform(3 ** -0.5)
+
+    policy = chainer.Sequential(
+        L.Linear(None, 400, initialW=winit),
+        F.relu,
+        L.Linear(None, 300, initialW=winit),
+        F.relu,
+        L.Linear(None, action_size, initialW=winit),
+        F.tanh,
+        chainerrl.distribution.ContinuousDeterministicDistribution,
+    )
+    policy_optimizer = optimizers.Adam().setup(policy)
+
+    def make_q_func_with_optimizer():
+        q_func = chainer.Sequential(
+            concat_obs_and_action,
+            L.Linear(None, 400, initialW=winit),
+            F.relu,
+            L.Linear(None, 300, initialW=winit),
+            F.relu,
+            L.Linear(None, 1, initialW=winit),
+        )
+        q_func_optimizer = optimizers.Adam().setup(q_func)
+        return q_func, q_func_optimizer
+
+    q_func1, q_func1_optimizer = make_q_func_with_optimizer()
+    q_func2, q_func2_optimizer = make_q_func_with_optimizer()
+
+    # Draw the computational graph and save it in the output directory.
+    fake_obs = chainer.Variable(
+        policy.xp.zeros_like(obs_space.low, dtype=np.float32)[None],
+        name='observation')
+    fake_action = chainer.Variable(
+        policy.xp.zeros_like(action_space.low, dtype=np.float32)[None],
+        name='action')
+    chainerrl.misc.draw_computational_graph(
+        [policy(fake_obs)], os.path.join(args.outdir, 'policy'))
+    chainerrl.misc.draw_computational_graph(
+        [q_func1(fake_obs, fake_action)], os.path.join(args.outdir, 'q_func1'))
+    chainerrl.misc.draw_computational_graph(
+        [q_func2(fake_obs, fake_action)], os.path.join(args.outdir, 'q_func2'))
+
+    rbuf = replay_buffer.ReplayBuffer(10 ** 6)
```

Comment:
Nit: Technically the replay buffer stores the entire history, which in this specific case is 10 ** 6. Maybe you could set this to `args.steps`, though not a major deal.

Reply:
They used 10**6. https://github.com/sfujim/TD3/blob/master/utils.py#L8